addons/sourcemod/scripting/VIP_SourcemodFlags.sp

```diff
@@ -268,6 +295,10 @@ stock void LoadClient(int client)
 			}
 
 			AdminInheritGroup(curAdm, grp);
+			if(newAdmin == true)
+			{
+				SetUserAdmin(client, curAdm, true);

```

Comment:
we already do `curAdm.BindIdentity(sAuthType, sAuth)` and inherit group flags+immunity so not needed